buildpipeline/DotNet-CoreClr-Trusted-Linux.json

```diff
@@ -49,7 +49,7 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "run --rm $(DockerCommonRunArgs) git clone $(GitHubRepo) $(GitHubDirectory)",
+        "arguments": "run --rm $(DockerCommonRunArgs) git clone $(PB_VsoCoreClrGitUrl) $(PB_GitDirectory)",
```

Comment:
Not sure the PB_ stuff is needed here as it's not the pattern for coreFX and never collides, rather it may be PB_ in the PB def but then passed in the globals like " VsoCoreClrGitUrl=$(PB_VsoCoreClrGitUrl)".   The fewer places we have PB_ the easier it gets when we move off it.